kubetest/process/process.go

```diff
@@ -170,9 +170,16 @@ func (c *Control) FinishRunning(cmd *exec.Cmd) error {
 		select {
 		case <-sigChannel:
 			log.Printf("Killing %v(%v) after receiving signal", stepName, -cmd.Process.Pid)
-			if err := syscall.Kill(-cmd.Process.Pid, syscall.SIGKILL); err != nil {
+
+			pgid, err := syscall.Getpgid(cmd.Process.Pid)
```

Comment:
Could you add a comment about why we need this?

Reply:
done